docusaurus/docs/reference/extensions-add-expose.md

```diff
@@ -0,0 +1,99 @@
+---
```

Comment:
Somewhere here we should clarify and link information on how to declare dependencies and declare the exposed components in the plugin.json